compiler/noirc_frontend/src/tests.rs

```diff
@@ -1740,22 +1748,26 @@ fn normal_generic_used_in_nested_array_length_fail() {
 fn numeric_generic_used_in_nested_type_pass() {
     // The order of these structs should not be changed to make sure
     // that we are accurately resolving all struct generics before struct fields
-    let src = r#"
+    assert_no_errors(
+        r#"
     struct NestedNumeric<let N: u32> {
         a: Field,
         b: InnerNumeric<N>
     }
     struct InnerNumeric<let N: u32> {
         inner: [u64; N],
     }    
-    "#;
-    let errors = get_program_errors(src);
-    assert!(errors.is_empty());
+    "#,
+    );
 }
 
 #[test]
 fn numeric_generic_used_in_trait() {
-    let src = r#"
```

Comment:
I do like having the `src` be separate and then passed into `assert_no_errors` but this is a nit and non-blocking